src/main/java/com/github/dockerjava/core/command/ListImagesCmdImpl.java

```diff
@@ -48,4 +49,22 @@ public ListImagesCmd withFilters(String filter) {
         return this;
     }
 
+    @Override
+    public String toString() {
+        return new StringBuilder("images ").append(showAll ? "--all=true" : "")
+                .append(filters != null ? "--filters " + filters : "")
+                .append(imageNameFilter != null ? "--filter " + imageNameFilter : "").toString();
+    }
```

Comment:
Just do toString from helpers (see some class), it was old idea to emulate command line.